src/imio/helpers/date.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+from DateTime import DateTime
+from imio.helpers.testing import IntegrationTestCase
+from zope.i18n import translate
+
+
+MONTHIDS = {1:  'month_jan', 2:  'month_feb', 3:  'month_mar', 4:  'month_apr',
+             5:  'month_may', 6:  'month_jun', 7:  'month_jul', 8:  'month_aug',
+             9:  'month_sep', 10: 'month_oct', 11: 'month_nov', 12: 'month_dec'}
+
+def formatDate(date, month_name=True, context=None, long_format=False):
+    """
+      Format the given date.  This is usefull for formatting dates in the pod templates
+      long_format : we want the hour with it
+      mont_name : we want the name of the month instead of the number
+      Note that we force european format...
+    """
+    if date == "now":
+        #we want the current date, calculate it
+        date = DateTime()
+
+    if long_format:
+        hour = ' (%H:%M)'
+    else:
+        hour = ''
+    #check if we need to return the month name
+    if month_name:
+        month = date.month()
+        translated_month = translate(msgid=MONTHIDS[month],
+                                     domain="plonelocales",
+                                     context=context,
+                                     default=MONTHIDS[month],
+                                     target_language='fr')
+        #use mOntH here to avoid strftime translating the month...
+        res = date.strftime('%e mOntH %Y' + hour)
+        #replace '01 ' by '1er '
+        res = res.replace('01 ', '1er ')
```

Comment:
Please manage day, month and year separately so you can easily replace day of '01' or '1' by '1er' without breaking dates like '2001' that could become '201er'...